CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ if(ROCKETS_FOUND AND ROCKETS_USE_LIBWEBSOCKETS)
     list(APPEND COMMON_FIND_PACKAGE_DEFINES BRAYNS_USE_NETWORKING)
   endif()
 else()
-  unset(BRAYNS_NETWORKING_ENABLED)
+  set(BRAYNS_NETWORKING_ENABLED OFF)
```

Comment:
`set(BRAYNS_NETWORKING_ENABLED)` should also work, doesn't it?

Reply:
Don't know, maybe? The explicit one is more obvious

Reply:
According to documentation it has the semantics as `unset`, hence the variable is uninitialized. But as it was set to `ON` before, this has no effect in practice, only an explicit set to `OFF`.